### DISCUSSION TITLE
User Control

### DISCUSSION THREAD

**Discussion by coderasm** [PrismLibrary/Prism#2373] [WPF]
If I want to create a reusable user control would It be better to build it without Prism? Are there pros/cons to building a reusable user control using vanilla wpf instead building it with prism? If I build It with prism, then it's lifecycle is tied directly to the prism container and I have to use the event aggregator instead of using bindings to dependency properties. Is it recommended to build the control completely with prism or completely without it?

**Comment by brianlagunas** [comment]
Do not build reusable UI controls with Prism.

**Reply by coderasm** [reply]
Thank you that is what I thought.
